config/routes.rb

```diff
@@ -1,7 +1,9 @@
 Rails.application.routes.draw do
+  get 'users/new'
```

Comment:
using " instead of ' and move it to scope locale

Reply:
done